web/api/formatters/rrd2json.c

```diff
@@ -217,8 +223,23 @@ int rrdset2anything_api_v1(
         , time_t *latest_timestamp
         , struct context_param *context_param_list
         , char *chart_label_key
+        , long stats
 ) {
 
+int i = -1;
+long step = 0x0000000000000001;
```

Comment:
Here you are assuming that Netdata will run only on 64 bits computer, but we have users using `32bit arm`, I would suggest you use the value 1 here instead the hex value.